src/ProjectModule/Components/ProjectList/ProjectList.tsx

```diff
@@ -1,7 +1,127 @@
-import React from 'react'
+import { useEffect, useState } from "react";
+import Modal from 'react-bootstrap/Modal';
+import Dropdown from 'react-bootstrap/Dropdown';
+import axios from "axios";
+import { baseUrl } from "../../../Constants/Components/Urls";
+import styles from "./ProjectList.module.css"
+import { useForm } from "react-hook-form";
+import noData from "../../../assets/noData.jpg"
 
 export default function ProjectList() {
+const[projects,setProjects]=useState([]);
+
+const {register,handleSubmit,setValue,formState:{errors}}=useForm();
+const[pageArray,setPageArray]=useState([]);
+
+const getAllProject=async(pageNo:any,pageSize:any,title:any)=>{
+try {
+  const token=localStorage.getItem("userToken");
+  const response=await axios.get(`${baseUrl}/Project/manager`,
+  {headers:{Authorization:token}
+  ,params:{pageSize:pageSize,
+    pageNumber:pageNo,
+  title:title}})
+
+  setPageArray(Array(response.data.totalNumberOfPages).fill().map((_,i)=>i+1));
+
+  console.log(response.data)
+  setProjects(response.data.data); 
+} catch (error) {
+  console.log(error)
+}
+}
+
+useEffect(()=>{ 
+getAllProject(1,5);
+},[])
+// ##################### Filtration ##############################
+const[titleSearch,setTitleSearch]=useState("");
+const getSearchByTitle=(data:any)=>{
```

Comment:
I suggest renaming this function to `searchByTitle` and the parameter that it take is the search Title not the event
`searchByTitle(title:string)`

and then change the handle on the `onChange={(e)=>searchByTitle(e.target.value)}`